change the title

tugberkugurlu opened the pull request:
since it talks about Azure Cosmos DB's implementation of the concept, rather than giving instructions on how to implement it which was the message conveyed by the previous title.

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
The syntax was incorrect, but the content is intended as a Note in our style. The correct syntax includes a space after the angle bracket, so it should be > [AZURE.NOTE]

Please feel free to edit your pull request and we will merge it. Thanks for your contribution.

gbellmann commented:
Hi @rmca14. I first thought of fixing the missing space after the angle bracket, as you noted, but when I checked the new version of the document (https://github.com/Azure/azure-content/blob/master/articles/hdinsight/hdinsight-provision-clusters.md) it has all the text as one [AZURE.NOTE] block, and not two as this document has. That's why I went the deletion way.

rmca14 commented:
@mumian 
Please review this PR and indicate here as a comment what the content should be. Thanks.

mumian commented:
The edit looks good to me.  Thanks.

tugberkugurlu committed:
change the title

since it talks about Azure Cosmos DB's implementation of the concept, rather than giving instructions on how to implement it which was the message conveyed by the previous title.
